client/app/views/wizard/main.html

```diff
@@ -154,7 +156,7 @@
       </div>
       <div class="col-md-6">
         <p class="wizard-help-hint">
-          <span data-translate>In order to deliver encrypted emails and files each recipient needs an associated PGP key. Cut and paste the key of the recipient or mark the checkbox:</span>
+          <span data-translate>In order to deliver encrypted emails and files each recipient needs an associated PGP key.</span>
```

Comment:
simplified the texts that was too messy; but this is an other story not related to your lovely work! ;)